scripts/gitlab/build_and_test.sh

```diff
@@ -38,12 +39,21 @@ then
 
     if [[ -d /dev/shm ]]
     then
-        prefix="/dev/shm/${hostname}/${spec// /_}"
+        if [[ -z ${job_unique_id} ]]; then
+          date_in_sec=$(date +%s)
+          while [[ -d $date_in_sec ]] ; do
```

Comment:
I don't think this is checking the correct location?

Reply:
Oops... of course! This explains why this was not working well even with the "sleep" thing.

Reply:
Fix, I think.